mars/services/task/supervisor/processor.py

```diff
@@ -456,3 +458,21 @@ def _finish(self):
 
     def is_done(self) -> bool:
         return self.done.is_set()
+
+    async def remove_func_storage(self):
+        tileables_need_clean_up_func = [
+            self.tileable_id_to_tileable[tileable_id]
+            for tileable_id in self._preprocessor.tileable_ids_need_clean_up_func
```

Comment:
After `func` cleaned up, does `tileable_ids_need_clean_up_func` need to be clear too?